docs/v1/payroll-uk/index.html

```diff
@@ -4928,7 +4935,8 @@ <h3>Usage and SDK Samples</h3>
     employee_leave_type = EmployeeLeaveType(
         leave_type_id = "00000000-0000-0000-0000-000000000000",
         schedule_of_accrual = schedule_of_accrual::BEGINNINGOFCALENDARYEAR,
-        opening_balance = 5.25)
+        opening_balance = 5.25,
+        schedule_of_accrual_date = )
```

Comment:
shouldn't there be some value for schedule_of_accrual_date?

Reply:
value is added